.gitignore

```diff
@@ -86,6 +87,7 @@ celerybeat-schedule
 # virtualenv
 venv/
 ENV/
+.venv*/
```

Comment:
what's used for/from?

Reply:
I use this convention and I thiiink it's fairly common in general

Reply:
/me whispers

https://julien.danjou.info/properly-managing-your-gitignore/

Reply:
fiiiine 😆